db/migrate/20160816150906_add_email_alert_signup_content_ids_to_topics.rb

```diff
@@ -0,0 +1,9 @@
+class AddEmailAlertSignupContentIdsToTopics < ActiveRecord::Migration
+  def change
+    add_column :topics, :email_alert_signup_content_id, :string
+
+    Topic.all.each do |topic|
```

Comment:
Using real models in migrations can be dangerous. `Topic` can change over time and callbacks and expected behaviour can change. This isn't a very risky one but it's generally good practise to either a) use raw SQL b) make a fake model. The fake model would look like..

```
class FakeTopic < ActiveRecord::Base
  self.table_name = "topics"
end

FakeTopic.all do |topic|
# etc
```

Here is another example from this app where we needed the associations too: https://github.com/alphagov/service-manual-publisher/blob/master/db/migrate/20160225130400_drop_content_owners.rb